org.code.toboggan.network/src/org/code/toboggan/network/request/extensions/project/NetworkProjectGetFiles.java

```diff
@@ -0,0 +1,76 @@
+package org.code.toboggan.network.request.extensions.project;
+
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.code.toboggan.core.extension.APIExtensionIDs;
+import org.code.toboggan.core.extension.AbstractExtensionManager;
+import org.code.toboggan.core.extension.ICoreExtension;
+import org.code.toboggan.core.extension.project.IProjectGetFilesExtension;
+import org.code.toboggan.network.WSService;
+import org.code.toboggan.network.request.extensionpoints.project.IProjectGetFilesResponse;
+import org.code.toboggan.network.request.extensionpoints.project.IProjectSubscribeResponse;
+import org.code.toboggan.network.request.extensions.NetworkExtensionManager;
+
+import clientcore.websocket.IRequestSendErrorHandler;
+import clientcore.websocket.WSManager;
+import clientcore.websocket.models.Request;
+import clientcore.websocket.models.requests.ProjectGetFilesRequest;
+import clientcore.websocket.models.responses.ProjectGetFilesResponse;
+
+public class NetworkProjectGetFiles implements IProjectGetFilesExtension, IProjectSubscribeResponse {
```

Comment:
Do we really want to getFiles() at the end of EVERY subscribe? [should this class directly implement the IProjectSubscribeResponse ?]